Sources/Apodini/Properties/Options/PropertyOption.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 
 /// A type erasure of a `PropertyOption`
 public struct AnyPropertyOption<PropertyNameSpace> {
-    /// A type erasued `PropertyOptionKey` identfying the `AnyPropertyOption`
+    /// A type erasued `PropertyOptionKey` identifying the `AnyPropertyOption`
```

Comment:
While you're at it 😄 
```suggestion
    /// A type erased `PropertyOptionKey` identifying the `AnyPropertyOption`
```

Reply:
Resolved via 6c2b9bf